views/templates/hook/back-office-header.tpl

```diff
@@ -0,0 +1,12 @@
+{*
+* 2014-2015 Stigmi
+*
+* @author Serge <serge@stigmi.eu>
+* @copyright 2014-2015 Stigmi
+* @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+*}
+<link rel="stylesheet" href="{$module_dir|escape}/views/css/admin-bpost.css" type="text/css" />
```

Comment:
Regarding comments, 1.22 is an intermediate version in which we added some features / options that are yet not visible (commented).

We'll uncomment / clean remaining old comments for 1.3.
Right now we need them to stay in the code.
Thx